app/assets/javascripts/browsered/web-payments/google-pay.js

```diff
@@ -13,6 +13,7 @@ const processPayment = paymentData => {
     body: JSON.stringify(paymentData)
   })
     .then(response => {
+      ga('send', 'event', 'Google Pay', 'Sucessful', 'auth/capture request')
```

Comment:
i take it this section of code handles authorisation and capture, and not just authorisation?

Reply:
Yeah it waits for the return from both bits